File: src/main/java/fr/ayfri/minecraft_art/buttons/SelectImageButton.java
package fr.ayfri.minecraft_art.buttons;

import fr.ayfri.minecraft_art.Main;

public class SelectImageButton extends Button {
	public SelectImageButton(final Main sketch) {
		super(sketch, sketch.width / 2, sketch.height - 100, 150, 80);
		setText("Select Image");
	}
	
	@Override
	public void onClick() {
		sketch.selectInput("Select an image.", "chooseFile");
	}
}

File: src/main/kotlin/io/github/ayfri/minecraft_art/buttons/SelectImageButton.kt
package io.github.ayfri.minecraft_art.buttons

import io.github.ayfri.minecraft_art.Main

class SelectImageButton(sketch: Main) : Button(sketch, sketch.width / 2, sketch.height - 100, 150, 80) {
	init {
		text = "Select Image"
	}

	override fun onClick() = sketch.selectInput("Select an image.", "chooseFile")
}
